evaporate.js

```diff
@@ -1372,7 +1372,7 @@
                     url = [con.signerUrl, '?to_sign=', stringToSignMethod(authRequester), '&datetime=', authRequester.dateString].join(''),
                     warnMsg;
 
-                var signParams = makeSignParamsObject(me.signParams);
+                var signParams = makeSignParamsObject(con.signParams);
```

Comment:
signParams is stored on the file because it needs to be restored from localStorage when re-using what's on S3.